### DISCUSSION TITLE
shell script failure

### DISCUSSION THREAD

**Discussion by haseeb-abdul9** [learndevopsonline/learndevopsonline#1798] [Q&A]
cart script fails halfway through
![Screenshot 2024-02-07 081823](https://github.com/learndevopsonline/learndevopsonline/assets/129119790/2298017d-0d13-49c3-9907-6410cd34a3a3)
![Screenshot 2024-02-07 081356](https://github.com/learndevopsonline/learndevopsonline/assets/129119790/e5cb1ab1-5695-46ca-8519-b9ad0b8809d1)
public IP: 34.229.188.249

**Comment by chaithanya1812** [comment]
HI, check names you given is diffrenet form zip & uzip commands.
you given /tmp/user.zip & uziping /tmp/cart.zip
![Screenshot 2024-02-07 191719](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/8720a66f-5e04-40c1-965f-48b32e8ebab3)
